src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/SolutionExplorerInProcess.cs

```diff
@@ -229,6 +230,10 @@ public async Task RestoreNuGetPackagesAsync(string projectName, CancellationToke
         public async Task SaveAllAsync(CancellationToken cancellationToken)
         {
             await TestServices.Shell.ExecuteCommandAsync(VSConstants.VSStd97CmdID.SaveSolution, cancellationToken);
+            await TestServices.Workspace.WaitForAllAsyncOperationsAsync(new[] { FeatureAttribute.Workspace }, cancellationToken);
```

Comment:
A comment here would be appreciated: it's odd to see this attached to the workspace since it's not actually the workspace being the origin of any asynchrony here. I'm not sure creating a new feature attribute is any _better_, but we have so many unnecessary waits on the Workspace sprinkled around that this might be culled at some point.

Reply:
➡️ Added comment